.drone.yml

```diff
@@ -866,6 +866,39 @@ steps:
   volumes:
   - name: docker
     path: /var/run/docker.sock
+- commands:
+  - ./bin/build export-version
+  depends_on:
+  - compile-build-cmd
+  image: grafana/build-container:1.7.7
+  name: export-version
+- commands:
+  - apk add --update --no-cache build-base make git coreutils bash docker gcc libc-dev
+  - export GRAFANA_IMAGE=grafana/grafana-enterprise-dev:$(cat $OSS_BUILD_VERSION_FILE)
+  - docker tag grafana/grafana-enterprise-image-tags:$(cat $OSS_BUILD_VERSION_FILE)-amd64
+    $$GRAFANA_IMAGE
+  - export E2E_NETWORK_NAME=e2e-$DRONE_BUILD_NUMBER-$DRONE_STAGE_NUMBER-$DRONE_STEP_NUMBER
+  - mkdir -p $HOME/.docker
+  - printenv GCR_CREDS > $HOME/.docker/config.json
```

Comment:
is there a risk of leaking these creds? i'm not sure if this is useful, but it looks like [this is how it's handled](https://github.com/grafana/grafana/blob/ec7a2476740f31ae02ca438e73b7767902dab64b/scripts/drone/events/cron.star#L26) in another pipeline

Reply:
good to know. will weigh this in on the other side (in enterprise)